neqo-transport/src/connection/mod.rs

```diff
@@ -1501,7 +1501,16 @@ impl Connection {
                         self.stats.borrow_mut().dups_rx += 1;
                     } else {
                         match self.process_packet(path, &payload, now) {
-                            Ok(migrate) => self.postprocess_packet(path, d, &packet, migrate, now),
+                            Ok(migrate) => {
+                                // Since we processed frames from this IP packet now,
+                                // update the ECN counts (RFC9000, Section 13.4.1).
+                                if let Some(space) = self.acks.get_mut(space) {
+                                    space.inc_ecn_count(d.tos().into());
+                                } else {
+                                    qdebug!("Not tracking ECN for dropped packet number space");
+                                }
```

Comment:
Is this the right place for the change?  postprocess_packet() seems better.

Reply:
Good idea, moved.